starboard/shared/starboard/player/player_worker.h

```diff
@@ -73,15 +75,16 @@ class PlayerWorker {
     typedef std::function<SbPlayerState()> GetPlayerStateCB;
     typedef std::function<void(SbPlayerState player_state)> UpdatePlayerStateCB;
     typedef std::function<void(SbPlayerError error,
+                               const HandlerResult& result,
```

Comment:
It seems that |result| doesn't convey any information, and is added to allow reusing `PlayerWorker::UpdatePlayerError()`.  Consider keeping the original UpdatePlayerErrorCB, and using std::bind() to remove the second parameter of `PlayerWorker::UpdatePlayerError()` when passing into the Handler as a parameter.